The Emerald Sword, an unique and ancient Sheragi warship. (#4786)
endless-sky/endless-sky: data/the emerald sword.txt

# Copyright (c) 2020 by Fernando S. Sáez
#
# Endless Sky is free software: you can redistribute it and/or modify it under the
# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
# PARTICULAR PURPOSE.  See the GNU General Public License for more details.

mission "The Box 1"
	landing

> 	name "The Sheragi bunker"

Author: Will apply this one manually for all missions later.

	description "Visit Albert Foster on Zug to see what he discovered."
	source
		near "Zubeneschamali" 1 100
	destination Zug
	
	to offer
		has "event: rim archaeology results"

> 		has "First Contact: Unfettered: declined"

Reviewer: ```suggestion
```
Is this event even possible? I couldn't find it in any of the game files and to proceed with the campaign, I had to delete this line from my local game files.

Author: This comes from the short mission that happens when you land on an Unfettered planet. There is some conversation, and some insight on them, and at the end of it, since it doesn't continue, the mission is declined. I placed this there as a requirement because that is the part that mentions the Sheragi, and makes it obvious that the Hai knows about them.

		has "main plot completed"
		random < 4
	
	on offer
		conversation
			`You receive a message from Albert Foster, the archaeologist who you helped uncover the mysterious Sheragi ruins on Zug. He seems to be very excited.`
			`	"<first>! It's been a while, huh? Remember the Sheragi dig site? I've just found something incredible here. Please, come visit me on Zug. I don't think I can trust anyone else with this."`
				accept

mission "The Box 2"
	landing

> 	name "The mysterious box"

Reviewer: I don't think that mission names should be lowercase, but there might be precedent for that that I can't recall. If there is, let me know, and if there isn't, I would recommend capitalizing all of the mission names.

	description "Take the mysterious box to a trusted scientist on <destination>."
	source "Zug"
	destination "Vinci"
	
	to offer
		has "The Box 1: done"
	
	on offer

> 		conversation

Reviewer: This conversation is so long that it actually goes off of my screen, which is somewhat problematic. I assume this is due to the fact that some parts of the code have been moved around incorrectly. I am attaching a screenshot at [this](https://drive.google.com/file/d/1RS7D1ERscLF0CZdbgpTRWaBoapFitTf4/view?usp=sharing) link so that you can see what I mean.

> 			`You approach to land near the dig site, and Albert is already waiting for you, watching your descent. he greets you as soon as the ship doors open.`

Reviewer: ```suggestion
			`You land at the dig site and notice that Albert is already waiting for you, watching your descent. He greets you as soon as the ship doors open.`
```

> 			`	"Hello, <first>! would you like to have a little tour of the museum?"`

Reviewer: ```suggestion
			`	"Hello, <first>! Would you like to go on a little tour of the museum?"`
```

			choice
				`	"Sure, I'm eager to see what you discovered."`
				`	"Just tell me what's so important."`
					goto hurry

> 			`	You two catch up as he leads you into the ruins, talking about the "minor" discoveries he made so far. He insists to keep the reason he brought you here a surprise. You enter one of the Sheragi buildings that have been uncovered, and there is a museum set up inside it, with various artifacts.`

Reviewer: ```suggestion
			`	You two catch up as he leads you into the ruins, talking about the "minor" discoveries he has made so far. He insists to keep the reason he brought you here a surprise. You enter one of the Sheragi buildings that have been uncovered to find that a museum has been set up inside it with various artifacts on display.`
```

> 			`	It's strange seeing so many technological devices as old as some of the oldest prehistorical tools on earth, most of them so deteriorated you wouldn't even be able to imagine what they were used for, if it wasn't for the descriptions in each of the displays. One of the most intact things is a small metal statue of a Sheragi in flight, possibly a trophy for some event, or merely a sculpture of a character. It's impossible to know, but it reveals their physical form. As you look at how tall the doors of this house is, and back at the statue, you realize how big they must have been. It's hard to imagine the size a spaceship must be to hope to fit a creature this large. After the tour is over, he leads you to one of the buildings that are off-limits to the general public.`

Reviewer: ```suggestion
			`	It's strange seeing so many technological devices as old as some of the prehistoric tools from ancient Earth, most of them so deteriorated you wouldn't even be able to imagine what they were used for if it wasn't for the descriptions in each of the displays. One of the most intact things is a small metal statue of a Sheragi in flight, possibly a trophy for some event, or merely a sculpture of a character. It's impossible to know, but it reveals their physical form. As you look at how tall the doors of this house are, and back at the statue, you realize how big they must have been. It's hard to imagine the size that a spaceship must have been to even hope to fit a creature this large. After the tour is over, he leads you to one of the buildings that are off-limits to the general public.`
```

			`	Albert opens the safe, and shows you a weathered metal cube, around half a metre on each side. He struggles with the weight, but manages to set it down on the floor.`
			`	"This appears to be a data storage device of some sort. It was connected to what we think were once computers, but nothing else is salvageable after spending thousands of years under wet dirt. Based on how sturdy this thing is, it's plausible that some of the data could have survived, like a time capsule."`
			`	"I have a contact back on Vinci who could probably figure this out, and this is where you come in. You remember John, right? I can't leave the dig site, but I also don't have anyone else I could trust to carry such a valuable artifact. Will you take it there for me?"`
			choice
				`	"Sure, I'll take the cube to him."`
					goto take

> 				`	"I'm sorry, but that sounds sketchier than all the other jobs I did for you. The quarg were clear in saying that some things are not meant to be found."`

Reviewer: ```suggestion
				`	"I'm sorry, but that sounds sketchier than all the other jobs I did for you. The Quarg were clear in saying that some things are not meant to be found."`
```

			label sketchy
			`	"What?" He seems incredulous.`
			`	"After going this far, you're abandoning me? Knowledge by itself isn't dangerous. Who knows what secrets this cube could hold?"`
			choice
				`	"Fine, I'll take it to him."`
					goto take
				`	"Sorry, but I'm not crossing that line."`
					decline
			label take
			`	"Splendid!"`

> 			`	You two put the cube in an inconspicuous backpack, and carry it back to your ship. Albert gives you an address where you'll deliver the cube, mentions that you will be paid after transporting it, and wishes you good luck on your travel.`

Reviewer: ```suggestion
			`	The two of you put the cube in an inconspicuous backpack and carry it back to your ship. Albert gives you an address where you'll deliver the cube, mentions that you will be paid after transporting it, and wishes you good luck on your travel.`
```

				accept
	
	on complete
		payment 500000
		event "cube stolen" 20 30
		log `Took a mysterious Sheragi data cube that Albert Foster dug up on Zug to a trusted friend on Vinci. John will take some time to see how exactly it works, and tell you when it's done.`
		conversation

> 			`You arrive on Vinci, and carry the cube to the address Foster gave you. John has been waiting for you.`

Reviewer: ```suggestion
			`You arrive on Vinci and carry the cube to the address Foster gave you. You find that John has been waiting for you.`
```

			`	"So, Foster is up to no good again, huh?"`
			`	He laughs as he lets you into his lab.`
			choice
				`	"It doesn't seem likely that we'll get in trouble this time."`
				`	"I'm slightly concerned myself, but I'm sure he knows what he's doing."`
			`	You set the backpack with the cube down on a table and open it up.`
			`	"Well, he's paying me to help with this, so no problem with me. It's not like everyone gets to see the exotic things he finds first hand."`
			`	John proceeds to examine the cube, picking it up and looking at it from different angles, trying to find some port to connect to. He finds one, but it's so deteriorated that any attempts to use it would be fruitless. He places the cube inside a machine to take a deeper scan of the object, and watches it silently working, staring patiently at the screen.`
			`	"Yes, this seems to be a data storage device of some sort... it's made to be sturdy, like a black box, so it's plausible that we could recover some data from it. This is going to take a very long time, though."`

> 			`	"How long?" You ask.`

Reviewer: ```suggestion
			`	"How long?" you ask.`
```

			`	"A couple months, probably. It depends on how alien this thing is, and if it's encrypted or not, but I should at least figure out if there is any recoverable data in here."`

> 			`	John says he'll drop you a message once his results come out, and you part ways and return to your ship.`

Author: What i wanted to mean by this is more along the lines of "yes, there are clearly ones and zeroes here."
Kind of how you can look at a CD in a microscope and confirm that it has data written in it by the "burned" lines, even if you're technically unable to read it.

mission "The Box 3"
	landing
	name "The box"
	description "Discover the whereabouts of the box from an information broker in the pirate world of Stormhold."
	source
		government "Republic" "Free Worlds" "Syndicate" "Independent" "Neutral"
		near "Sol" 100
		attributes spaceport
	destination "Stormhold"
	
	to offer
		has "event: cube stolen"
	
	on offer
		conversation
			`You receive a call from John, who has been working on the mysterious Sheragi black box for about a month now.`
			`	"Captain <last>! I have some bad news and some worse news. Which one do you want to hear first?"`
			choice
				`	"The bad news."`
					goto bad
				`	"The worse news."`
					goto worse
			label bad
			`	"The bad news is that this is a lot more complicated than I thought. I found out that the data in the box has survived, and can still be recovered after all this time, but it will need some specialized hardware and knowledge that we don't have yet."`

> 			`	"And the worse news?" You ask.`

Reviewer: ```suggestion
			`	"And the worse news?" you ask.`
```

> 			`	Well... Some people invaded my lab during the night, and stole a lot of the things I was working on. Thankfully my insurance covers most of it, and I still have most of my heavy expensive equipment, but the box... is gone.`

Reviewer: ```suggestion
			`	"Well... Some people invaded my lab during the night and stole a lot of the things I was working on. Thankfully my insurance covers most of it, and I still have most of my heavy expensive equipment, but the box... is gone."`
```

				goto whatnow
			label worse

> 			`	"Well... Some people invaded my lab during the night, and stole a lot of the things I was working on. Thankfully my insurance covers most of it, and I still have most of my heavy expensive equipment, but the box... is gone.`

Reviewer: ```suggestion
			`	"Well... Some people invaded my lab during the night, and stole a lot of the things I was working on. Thankfully my insurance covers most of it, and I still have most of my heavy expensive equipment, but the box... is gone."`
```

			`	"And the bad news?"`
			`	"The bad news is that this is a lot more complicated than I thought. I found out that the data in the box has survived, and can still be recovered after all this time, but it will need some specialized hardware and knowledge that we don't have yet."`
			label whatnow
			`	"What do we do now? Do you have any idea where to find the box? Did you contact law enforcement?"`

> 			`	"I did what I could, but if this was mandated as I expect, the police isn't going to help, and the box is probably off-world by now. However... if you have some cash at hand, and you're willing to go the extra mile to recover the box, there is a way."`

Reviewer: ```suggestion
			`	"I did what I could, but if this was mandated as I expect, the police aren't going to help, and the box is probably off-world by now. However... if you have some cash at hand, and you're willing to go the extra mile to recover the box, there is a way."`
```

			choice
				`	"What do you mean?"`
					goto info
				`	"Are you trying to extort me?"`

> 			`	"What? No, of course not! Do you think I would steal my own work? Let me explain...`

Reviewer: ```suggestion
			`	"What? No, of course not! Do you think I would steal my own work? Let me explain..."`
```

			label info

> 			`	"Albert's methods are... nonstandard, to say the least, and we've dealt with a certain individual once. An information dealer, who lives in Stormhold. Sketchy person, seems to have informants nearly everywhere, and knows about everything. They're known as Mammon. If you want to find the box, they'll probably know where it is."`

Reviewer: ```suggestion
			`	"Albert's methods are... nonstandard, to say the least, and we've dealt with a certain individual once. An information dealer who lives in Stormhold. Sketchy person, seems to have informants nearly everywhere and knows about everything. The individual in question is known as Mammon. If you want to find the box, dealing with Mammon is probably your best bet."`
```
I understand that the use of the word is to prevent the implication of any genders, but it felt somewhat awkward to me so I felt this was a better way to phrase it. Feel free to change that up if you don't like it.

			choice
				`	"Thank you, I'll see if I can find this Mammon person."`
				`	"I'm sorry, but I think nothing good can come out of this. Better to forget about the cube."`
					decline

> 			`	"Knowing their methods, they'll likely find you first. But don't be alarmed, they're not... evil, per se. Just be careful, alright?"`

Reviewer: ```suggestion
			`	"Trust me, Mammon will definitely find you first. Even so, don't be alarmed; Mammon isn't... evil, per se. Just be careful, alright?"`
```

				accept
	
	to complete
		never

mission "The Box 4"
	landing
	name "The box"
	description "Confront the mercenaries who stole the cube on Watcher."
	source "Stormhold"
	destination "Watcher"
	
	to offer
		has "The Box 3: offered"
	
	on offer
		conversation
			branch visited
				has "visited mammon"
			branch first
				not "visited mammon"
			label first

> 			`You arrive safely on Stormhold, despite the pirates clearly not wanting you here. You're still halfway through the atmosphere, and someone hails you, with a robotic, modulated voice.`

Reviewer: ```suggestion
			`Despite the obvious hostility the locals display towards you, you arrive safely on Stormhold. You're still halfway through the atmosphere when a robotic, modulated voice hails you.`
```

			`	"I heard you were looking for me, <first> <last>," the person on the other end says.`
			choice
				`	"Who are you? How do you know my name?"`
					goto who
				`	"You did? How?"`

> 			`	"In my line of business, I have to know everything. Including all of my clients. How else would you hope to find me?"`	

Reviewer: ```suggestion
			`	"In my line of business, I have to know everything. That includes the identities of any potential clients, such as yourself. How else would you hope to find me?"`	
```

				goto intro
			label who
			`	The mysterious person laughs.`
			`	"It looks like I know you more than you know yourself. I am who you're looking for, of course."`
			label intro

> 			`	"Mammon. Information dealer. At your service." The voice speaks ominously.`

Reviewer: ```suggestion
			`	"Mammon, information vendor. At your service," the voice states ominously.`
```

			`	"Come meet me in person to discuss the specifics of what you're looking for."`

> 			`	The connection cuts, and you receive coordinates to somewhere apparently random, in the middle of the sea.`

Reviewer: ```suggestion
			`	The connection cuts and you receive coordinates to a seemingly random location that appears to be in the middle of the sea.`
```

> 			`	You arrive at the proposed location in the middle of nowhere, under stormy conditions. Initially, you question whether you received the right coordinates or not, until a submarine emerges from the waves. It is a huge vessel, and lights guide you to land on it's deck, which your spaceship comfortably fits in. Armed men in business suits surround your ship, and when you unboard, two of them guide you inside. You're led into a dimly lit room with a holographic map of the galaxy on the ceiling and a desk, behind which a hooded figure sits.`

Reviewer: ```suggestion
			`	You arrive at the proposed location in the middle of nowhere under stormy conditions. You begin to question whether you received the correct coordinates until a massive submarine emerges from the waves. It is a gargantuan vessel, and as lights guide you to land on its deck, you notice that your spaceship comfortably fits on the deck despite the size of your vessel. Armed men in business suits surround your ship and once you disembark, two of them guide you inside. You're led into a dimly lit room with a holographic map of the galaxy on the ceiling and a desk, behind which a hooded figure sits.`
```

> 			`	"Please, take a seat," they say. Their voice has a strange tone with no distinguishable gender, with a sybilating, guttural accent, and it's not until you take a seat that you notice a reptilian snout poking out of the hood. You don't know whether this is natural or an extreme body modification.`

Reviewer: ```suggestion
			`	"Please, take a seat," they say. Their voice has a strange tone with no distinguishable gender, with a sibilating, guttural accent, and it's not until you take a seat that you notice a reptilian snout poking out of the hood. You don't know whether this is natural or an extreme body modification.`
```

			choice
				`(Say nothing.)`
					goto middle
				`"Are you a..."`
			`	They cut you off. "Who or what I am doesn't matter to you."`
			label middle
			`	"Let's get to what matters. You want to find that alien artifact, yes?"`
			`	You nod in response.`
			`	"I have the information you need. Finding lost objects is the most trivial of my services, but as with everything, it has a price.`
			`	"Ten million credits." they say. "Non-negotiable. And don't try anything funny."`
			branch cash

> 				credits >= 12000000

Author: I left this with more than the real amount because the payment is made elsewhere after the mission. This means the player could arrive with exactly 10M, accept the quest, and with mortgage or crew payments, go under 10M and avoid the payment quest. It would of course still be offered after the player has enough money again and take it all, but it could be months and would feel weird. I'm unable to put the payment in this quest since it needs to still be offered if the player doesn't have the money, and if i did that, it wouldn't get offered unless the player had more than 10M. So I worked around the code limitations.

> 				credits >= 12000000

Reviewer 2: Given that the payment *is* in a separate mission, I'd say the dialog from Mammon needs to change up a bit. Right now you give them the credit chips, then they give you the information, making it seem as though they accepted them, but then you later receive the mission saying the credits were withdrawn, which seems very confusing. 

How about making it so that Mammon declines the credit chips that you try to give to them, instead saying "I will make my withdrawal when I desire. I only required that you accept paying," or something to that effect. Then, the payment mission could perhaps only trigger at a random point after this mission is over and the player has >12 million, but the player can accept to pay even if that have only >10 million. In that way a player with >10 million can accept, but they won't be made broke because of the >12 million trigger for the 10 mil to be withdrawn.

> 				credits >= 12000000

Author: Interesting idea. This also makes mammon seem to be even more influential than now. I'll try to do that.

					goto haggle
				`	"How about you tell me what I want and I don't blow up your ship and your pirate friends?"`
					goto threat
			label cash
			choice
				`	"I need to think about this. Give me some time."`
					defer
				`	"That's a lot of money just for information. Can you make it cheaper?"`
					goto haggle
				`	"How about you tell me what I want and I don't blow up your ship and your pirate friends?"`
					goto threat
				`	"Here's the money."`
					goto pay
			label nomoney
			`	"I know you don't. I can arrange for the Payment to be made in different ways. Organ donation, prostitution, volunteering as a test subject... We'll need an appraisal first though."`
			`	You politely decline.`
			`	"Come back to me when you have it, then, and I shall provide you with the information. We have all the time in the universe. Until you die, or someone decides to destroy the cube, that is."`
			`	They gesture, and the men promptly and diligently escort you back out to your ship.`
				defer
			label haggle
			`	Mammon doesn't look pleased.`
			`	"What part of non-negotiable did you not understand? You're pushing your luck, captain <last>."`

> 				goto question

Reviewer: This allows an infinite loop to be formed where the player can continue haggling with no effect. I'd recommend changing it so that you can haggle once and then have to make an actual decision.

Author: Good idea, i'll try this.

			label threat
			`	Mammon just smiles, not at all intimidated.`
			`	"You think you can intimidate me? It's your funeral, human. If you were smart, you would know that my power far exceeds yours. You know I can't let you leave here alive now. It would set a bad precedent for me. Safety is my top priority."`
			`	You look around, and the men in suit are pointing guns at you. Alien guns. Before you can react, Mammon shoots you from under the table, fatally injuring your abdomen. Your half conscious body is dragged out of the ship, and your ship being shredded to pieces by alien weapons is the last thing you see.`
				die
			label pay
			`	Mammon takes the credit chips, examining them.`

> 			`	"Very good. The artifact you're looking for has been taken by some mercenaries. It seems to me that your friend messed with the wrong people. But that is not the information you paid for."`

Reviewer: ```suggestion
			`	"Very good. The artifact you're looking for has been taken by some mercenaries. It seems to me that your friend messed with the wrong people, but I digress. That is not the information you paid for."`
```

> 			`	They lift their hands to the holographic map of the galaxy in the ceiling. It zooms in, filling up the entire room with stars. In front of you, you recognize the border betwen the Deep and the Dirt Belt.`

Reviewer: ```suggestion
			`	Mammon lifts their hands to the holographic map of the galaxy in the ceiling. It zooms in, filling up the entire room with stars. In front of you, you recognize the border betwen the Deep and the Dirt Belt.`
```
The “their” in this sentence still sounds a bit strange, but I couldn’t think of any other ways to get the message across without removing that word. If you can come up with something better, I’m all for it.

> 			`	"They have kept the artifact for themselves to sell on the Black market, and their base of operations is right here in Algieba."`

Reviewer: ```suggestion
			`	"They have kept the artifact for themselves to sell on the black market, and their base of operations is right here in Algieba."`
```

> 			`	The hologram zooms further in, on Watcher, and they point at it.`

Reviewer: ```suggestion
			`	The hologram zooms further in on the moon Watcher and continues until it finally stops on a large forest.`
```

> 			`	"You will find them in this uninhabited moon. They are hiding out in this forest, and the artifact is with them. Expect hostilities."`

Reviewer: ```suggestion
			`	"You will find them on this uninhabited moon. They are hiding out in this forest, and the artifact is with them. Expect hostilities."`
```

			`	You thank Mammon for the service and get up to leave.`

> 			`	"Farewell, captain <last>. Remember: I am always watching."`

Reviewer: ```suggestion
			`	"Farewell, Captain <last>. Remember: I am always watching."`
```

				accept
	
	on defer
		set "visited mammon"
	
	on accept
		set "paid mammon"
		fail "The Box 3"

> 

Reviewer: Once you complete this mission, there are two "The box" missions in your mission list. I am not sure where this issue comes from or how to fix it, so I would appreciate it if someone who knows why this happens would explain in a comment.

mission "Mammon Payment"        
	name "Mammon Payment"        
	landing
	invisible

	to offer
		has "paid mammon"
		credits >= 10000000

	on offer
		payment -10000000
		conversation     
			`You receive a message informing you of a very large transaction: 10 million credits. Mammon's crew have withdrawn the money you paid for the information about the box.`
				decline

mission "The Box 5"
	landing
	name "The box"
	description "Destroy the mercenary fleet."
	source "Watcher"
	destination "Watcher"
	
	to offer
		has "The Box 4: done"
	
	on offer
		conversation

> 			`You arrive on the uncolonized world, and scout around the area Mammon pointed out. It's relatively easy to find some smoke rising from a campfire in a clearing, but as you approach to investigate, you're surprised to find some large ships taking off to greet you, ditching the green tarps used to hide among the trees. Looks like they really don't want you here. It's a fight!`

Reviewer: ```suggestion
			`You arrive on the uninhabited moon of Watcher and scout the area Mammon pointed out. It's relatively easy to find some smoke rising from a campfire in a clearing, but as you approach to investigate, you're surprised to find some large ships taking off to greet you, ditching the green tarps used to hide among the trees. Looks like they really don't want you here. It's a fight!`
```

		dialog
			`You managed to destroy the entire mercenary fleet. You should have no problem landing on their camp now and recovering the cube.`
	
	on visit
		conversation
			`You attempt to evade the mercenary ships and make a quick assault on their encampment, but the ships follow you through the atmosphere, and ground personnel start shooting at you. Bombing it on a pass means you risk destroying the cube. You have no choice but to go back to space and eliminate the mercenary fleet.`
				launch
	
mission "The Box 6a"
	landing
	name "The box"
	source "Watcher"
	destination "Vinci"
	invisible
	
	to offer
		has "The Box 5: done"
	
	on offer
		conversation
			`You come in for a landing, and the few mercenaries that stayed on the ground march to your ship carrying a makeshift white flag. Dozens of men and women, tough looking and in uniform, unarmed, remaining peaceful and organized. These weren't ordinary criminals, it seems.`
			`	"We know when we've lost," one of them yells to your ship on a megaphone. "What is it that you want?"`
			choice
				`(Shoot them.)`
					goto shoot
				`(Talk to them.)`
			
			`	You land and cautiously open the hatch to talk with them.`

> 			`	"Who do you work for? Where is the cube you stole?" You ask.`

Reviewer: ```suggestion
			`	"Who do you work for? Where is the cube you stole?" you ask.`
```

				decline
			
			label spare
			`	"Deal. Bring me the cube," you say.`
			`	While one of them goes fetch it, you wait in an uncomfortable silence, watching their every move, while they watch yours. Eventually the young man comes back and hands the cube to you. You examine it, confirming it is the artifact you were looking for.`
			`	"You have it, now go away. We won't get in your way again," she says, and the other mercenaries relax, going back to their tents.`
			`	With the cube in hand, you have no more reason to stay here. You go back to your ship, ready to take it back to John.`
				accept
	
	on decline
		payment 853621

mission "The Box 6b"
	landing
	name "The box"
	description "Bring the cube back to John on Vinci."
	source "Watcher"
	destination "Vinci"
	
	to offer
		has "The Box 5: done"
	
	on complete
		conversation
			`Carrying the mysterious box again, you make your way to John's lab. He lets you in, and you tell him the whole ballad of the lost cube.`
			`	"So, retribution, huh? You think it was the officials from the Deep that got angry at us last time? Or maybe Foster's competitors for that dig site? I should have seen that coming."`

> 			`	"Possibly" you say. I can't imagine any other reason."`

Reviewer: ```suggestion
			`	"Possibly," you say. "I can't imagine any other reason."`
```

			`	"At least you got it back, but I don't know if it would be wise to continue to work on it here, and I can't just leave the planet for a couple months with no consequences. I don't even have a clue on how to read that data yet."`
			`	"And what, do you happen to have another shady contact who can help?" you ask.`
			`	"Unfortunately not. And now we seem to be running against time. If only the aliens would be willing to work with us and give us the information we need to crack this box..."`
			`	As soon as you hear that phrase, it suddenly hits you. You've met the Hai up north, and you heard from the Unfettered that they've met the Sheragi many millenia ago. Perhaps the Hai could be able to help.`
			`	"I think I know what to do. You're not the only person with contacts in strange places."`
			`	You pick up your things, placing the cube back inside the backpack.`
			`	"Well, good luck with that, captain. Don't get in too much trouble."`
		
mission "The box 7a"
	landing
	name "The box"

> 	description "Travel to Mirrorlake to visit the Hai museum of galactic history."

Author: It's a good idea, but as of right now i have no idea how to execute it. I'll keep it in mind though!

	source

> 		near "Ya Hai" 1 100

Reviewer: Maybe you could make two separate missions with the same content but one of them checks if you have met the Hai and if so sends you directly there whereas the other is the one used here to send the player exploring if they haven't met the Hai. This way you still send the player who doesn't know the Hai exploring but the player who already does know them can just go straight there. It's a small change, but I feel it would be beneficial to the overall experience.

Author: Part of the mission relies on not only meeting the Hai, but also having done first contact with the Unfettered. That's the requirement for the questline to start. Since by that time in the mission chain it's implied that there are time constraints, i think it wouldn't fit it to just go exploring, it would require some major changes.

		attributes "hai"
	destination "Mirrorlake"
	clearance
	
	to offer
		has "The Box 6b: done"
	
	on offer
		conversation

> 			`	You ask around, and manage to find a Hai tourist guide around the spaceport. You ask him if there's any place where you could find more information about the Sheragi.`

Reviewer: ```suggestion
			`	You ask around and manage to find a Hai tourist guide around the spaceport. You ask him if there's any place where you could find more information about the Sheragi.`
```

> 			`	"A Human knows about them? I'm surprised. This is ancient history. If there's any place where you can find out more about them, it's the museum of galactic history in Mirrorlake."`

Reviewer: ```suggestion
			`	"A human knows about them? I'm surprised. This is ancient history. If there's any place where you can find out more about them, it's the museum of galactic history in Mirrorlake."`
```

			`	You thank the guide and go back to make your preparations to visit it.`
				accept

mission "The box 7b"
	description "Land on Hai-home to speak with the ministry of culture."
	landing
	name "The box"
	source "Mirrorlake"
	destination "Hai-home"
	clearance
	passengers 1
	
	to offer
		has "The box 7a: offered"
	
	on offer
		fail "The box 7a"
		conversation

> 			`Arriving in Mirrorlake, you ask around a little bit, and eventually someone directs you to the museum of galactic history; a major, but surprisingly not very popular touristic spot. Most Hai aren't very fond of looking at the past, and most Humans have no idea this place exists. The museum staff are surprised to see a Human face, but they let you wander regardless.`

Reviewer: ```suggestion
			`Arriving in Mirrorlake, you ask around a little bit, and eventually, someone directs you to the museum of galactic history; a major, but surprisingly not very popular touristic spot. Most Hai aren't very fond of looking at the past, and most humans have no idea this place exists. The museum staff is surprised to see a human face, but they let you wander regardless.`
```

			`	"With the proper tools, we could decode it. But in return, we'd like to have a copy of the contents as well."`
			`	You ponder your choices for a moment. There is no way you could solve this puzzle without their help. You've went this far, so you agree.`
			`	"Excellent!"`
			`	They chat with you a bit more, and the leader excuses himself for a moment to make a call. A moment later he returns.`
			`	"We have arranged for a very capable team of archaeologists to meet you in the spaceport in a few hours. They specialize in Sheragi artifacts. I advise you to make room for 3 passengers and some cargo for their equipment."`
			`	You thank them for their help, and leave the building, ready to meet your new team.`

> mission "The box 8"

Reviewer 2: Another mission title that needs capitalized.

> 	description "Take the Hai archaeologists to meet Albert Foster on Zug."

Reviewer 2: ```suggestion
	description "Take the Hai archaeologists to meet Albert Foster on <destination>."
```

	name "The box"
	source "Hai-home"
	destination "Zug"
	passengers 3
	cargo "Archaeological Equipment" 1
	
	to offer
		has "The box 7b: done"
	
	blocked `You arrive at the spaceport, but then you realize your ships don't have enough free bunks or cargo space required to bring the archaeologist team with you. You manage to contact them, and in an embarrassing conversation ask them to wait for you while you make sure you're ready.`
	
	on offer
		conversation
			`You meet up with the Hai archaeologists that will accompany you to human space, an all female team. They introduce themselves to you as Eyama, Rakei, and Desyo.`

> 			`	Eyama is a linguist, specializing in the Sheragi language. She playfully greets you in their rather guttural tongue, with strange tones.`

Reviewer 2: ```suggestion
			`	Eyama is a linguist specializing in the Sheragi language. She playfully greets you in their rather guttural tongue, with strange tones.`
```
I mentioned earlier how Foster could have made some mention of having translated some of the Sheragi language. Having a character that actually knows the language though could lead to some interesting banter. "Actually, this is all wrong."

			`	Rakei is an archaeological engineer, who immediately takes a liking to the cube as soon as she sees it, examining it's structure.`

> 			`	Finally, Desyo is a xenoprogramming expert, who studies alien software, and seemingly has a lot of experience with Sheragi code. Apparently, the displays of the Sheragi room of the museum all run in this exotic language, unseen by the usual public.`

Reviewer 2: ```suggestion
			`	Finally, Desyo is a xenoprogramming expert who studies alien software, and seemingly has a lot of experience with Sheragi code. Apparently, the displays of the Sheragi room of the museum all run in this exotic language, unseen by the usual public.`
```

			`	After the brief introduction, they each take their equipment and load them on the cargo bay of your ship. Rakei approaches you after everything is in place.`

> 			`	"We want to meet the human archaeologist who uncovered the ruins in the rim. Can you take us there? We will start examining the cube along the way. I think by the time we get there, we'll have something to work with."`

Reviewer 2: ```suggestion
			`	"We want to meet the human archaeologist who uncovered the ruins in the Rim. Can you take us there? We will start examining the cube along the way. I think by the time we get there, we'll have something to work with."`
```

			`It was a long trip, almost halfway across the galaxy. The Hai archaeologists have been friendly to you, giving you updates every time they make a new breakthrough. During this time, the cube, previously thought to be impenetrable, had been disassembled and cleaned. Components that decayed into a mess of oxidized wires and goopy paste over the eons were been identified and removed to substitute for equivalent ones, and a blueprint of the box and it's circuits has started to take shape.`
			`	You land, and make a quick call to Albert Foster, who doesn't believe you when you say you got alien archaeologists to help. You then look for the three Hai to figure out how they're going to go unnoticed in a Human world. You're surprised to find Eyama wearing long, dark but colorful baggy Human clothing with an equally oversized hoodie and a bandanna. You recognize this outfit, very popular with the youth these days, whom are generally regarded as delinquents by the previous generation. Additionally she's carrying two more sets, for her Hai partners as well. It hides her body really well at the cost of making her look like a young boy, and the Hai's relatively short stature doesn't help.`

> 			`	"Call me Miss Deus ex Machina, because I thought ahead and bought these from a merchant back home. Perfect disguise. The humans will never notice. How do I look?"`

Author: It's an intentional hint that Eyama is savvy on human language and culture as well. I dropped a few of these without directly stating it

> 			`	"Call me Miss Deus ex Machina, because I thought ahead and bought these from a merchant back home. Perfect disguise. The humans will never notice. How do I look?"`

Reviewer 2: It's cheeky. I like it.

> 			`	"Call me Miss Deus ex Machina, because I thought ahead and bought these from a merchant back home. Perfect disguise. The humans will never notice. How do I look?"`

Reviewer: I figured. My only concern was that the lack of other references to her extensive knowledge of humans throughout the rest of the story made this one feel somewhat out of place, although I might be remembering incorrectly.

> 			`	You can't disagree that it makes a good disguise, but refrain from telling them exactly how silly they would look in that clothing. Meanwhile, the other two go back to their rooms to get dressed.`

Reviewer 2: Seems like there's a lot of text here; should probably add a choice after Eyama asks how she looks.

			`	"Can you take us to Mr. Foster's museum? We'll take the cube and fill him in on the details."`
			`	You agree, and after they're all ready, you make your way out to the dig site. Surprisingly, the three Hai don't get many weird looks if they walk a bit further behind you. You arrive at the museum and they immediately break character by being very obviously excited to see the exhibits. This probably comes as a relief to onlookers, however, who would normally expect that type of kid to vandalize priceless artifacts.`

> 			`	Finally, you arrive at Albert's workshop. He stops brushing what looks like what was once a television and looks startled at you and the three "kids". The Hai take off their hood and bandanna, causing his jaw to nearly drop to the floor.`

Reviewer 2: ```suggestion
			`	Finally, you arrive at Albert's workshop. He stops brushing what looks like what was once a television and looks startled at you and the three "kids." The Hai take off their hood and bandanna, causing his jaw to nearly drop to the floor.`
```

			`	"I'll be damned. The Hai really exist!" He exclaims.`

> 			`	"Of course we do" Eyama laughs. "Haven't you heard all the stories?"`

Reviewer 2: ```suggestion
			`	"Of course we do," Eyama laughs. "Haven't you heard all the stories?"`
```

			`	"I had my doubts. Merchants are well known for telling tall tales." He smiles. "Anyway, I'm Albert Foster, archaeologist, pleased to meet you all."`

> 			`	They introduce themselves, and very soon the conversation evolves to a kind of jargon even Albert has trouble following. They talk about the box, and their findings, while Albert talks about the history uncovered in the dig site, and insights on the Sheragi daily life and culture. You close the door to make sure nobody else is startled.`

Reviewer 2: ```suggestion
			`	They introduce themselves, and very soon the conversation evolves to a kind of jargon even Albert has trouble following. They talk about the box and their findings, while Albert talks about the history uncovered in the dig site and insights on the Sheragi daily life and culture. You close the door to make sure nobody else is startled.`
```

			`	"We've managed to open the box and we're making a blueprint of it," says Desyo. "Once that's done, we might be able to rebuild the broken pieces and recover the data. We already have documentation on how their hardware worked."`
			`	"The catch is that this is a collaboration now. We'll help you, if you share your findings with us," Rakei continues.`

> 			`	"Sounds like a win-win to me. We'll arrange the details later. <first>, thank you so much for your help. We'll continue studying this artifact, and we'll send you a message when we're done."`

Reviewer 2: ```suggestion
			`	"Sounds like a win-win to me," Albert says. "We'll arrange the details later. <first>, thank you so much for your help. We'll continue studying this artifact, and we'll send you a message when we're done."`
```

			`	You say goodbye to Albert, Rakei, Eyama, and Desyo, and make your way back to your ship.` 
		event "cube decoded" 40 50
		
mission "The Emerald Sword 1"
	description "Visit Albert Foster on Zug."
	landing
	name "Black box decoded"
	source
		near "Zubeneschamali" 1 100
	destination "Zug"
	
	to offer
		has "event: cube decoded"
	
	on offer
		conversation
			`You're still in the middle of the post-flight check when you receive a new message from Albert Foster.`
			`	"Hello <first>. I have some good news! We have decoded what's in the black box. You should come visit us on Zug; we have plenty to tell you!"`
				accept

mission "The Emerald Sword 2"
	landing
	description "Bring an additional Jump Drive to Zug."
	name "The Emerald Sword"
	source "Zug"
	destination "Zug"
	
	to offer
		has "event: cube decoded"
		has "The Emerald Sword 1: done"
	
	on offer

> 		require "Jump Drive"

Author: Should i remove the Jump Drive requirement? I don't think a "blocked" text would fit in here, but this mission is post-main-quest, so the player should theoretically have a JD. Maybe i'll make the question afterwards a choice

> 		require "Jump Drive"

Reviewer 2: It can be pretty reasonably assumed that the player does have a JD at this point, but we can't really guarantee that. As such, it's either a matter of removing this requirement, or adding this requirement to the previous mission.

> 		require "Jump Drive"

Author: I changed the choice to fix this problem.

		conversation
			`You arrive on Zug, and after contacting them and following their instructions, you find Albert and the three Hai archaeologists in an ice cream shop. They're still wearing the youngster clothes, but not even bothering to hide their faces with the bandanna anymore. Surprisingly, nobody seems too distressed at the sight. Rakei waves for you and stands up, coming to you with ice creams in paw.`
			`	"Hello <first>! It seems our disguise worked better than intended. The humans around here think we're wearing a costume," she says quietly.`
			`	"Apparently young humans like to dress up as aliens?" Eyama responds confusedly.`
			`	"Yarthis saw right through it, though," Foster adds. "But oddly, the Quarg haven't given us much trouble."`
			`	"It has been an interesting cultural exchange. Human space might be a great vacation spot for us. But we're not on vacation yet. We have a mystery to solve," Eyama continues.`
			`	You sit down with your group on a bench in a nearby park, mostly empty, and Desyo picks up the conversation.`
			`	"So, the files in the black box were thankfully standard for Sheragi. Not very difficult to decode, once we managed to read them."`

> 			`	"As we told you in the message, it was all fleet movement data and military strategy details. This system was apparently on the border of one of the faction's territories. On the final moments of the recording, a large transport fleet had entered the system, unarmed ships. They were followed by military vessels in pursuit. There's no telling what they thought at the time, but a military fleet was immediately dispatched to eliminate both fleets. One particular ship, which Eyama has translated the name as 'Emerald Sword', started behaving erratically when given the orders, fired and missed a few shots, until it eventually stopped and fled. Clear signs of a mutiny. However, that ship fled to a specific direction, away from any hyperlanes."`

Author: Whoops, that's from an earlier version of the quest. I'll adjust that.

			`You have a jump drive in your possession. Would you like to give it to Albert's crew?`
			choice
				`    (No.)`
					defer
				`    (Yes.)`
			`   You give Albert a call, and arrange to meet with him to plan the next step. He shows up at your ship, greets you, and starts loading in some equipment along with the jump drive you got. He has assembled a crew of around 20 people, including him and the three Hai archaeologists, now comfortable to let go of their disguise once they're back in your ship.`
			`	"<first>, meet the people who will help us recover the Emerald Sword. I've just briefed them on the situation and what we plan to do. We're going to take some specialized tools with us, along with the Jump Drive, that will allow us to repair and outfit the ship in any world we can land on. I suggest you also hire some additional crew to man the ship once we capture it, since it's not going to pilot itself. If you're ready to go, Here are the coordinates to Zubenelhakrabi."`
			`	You thank Albert, and make your preparations for the voyage into the unknown.`
				accept
	
	on accept
		fail "The Emerald Sword 2"
		outfit "Jump Drive" -1
		event "zubenelhakrabi discovered"
	

> 	on enter "Zubenelhakrabi"

Author: Not without the use of a timer ship, and with that, i think the delay would be too long. There's a precedent for these long sections on arrival anyway, the same thing happens on Sayaiban.

		conversation

> 			`The trip to Zubenelhakrabi is completely normal and calm. Given everyone's trepidating expectations though, the trip's uneventfulness only manages to keep everyone on edge. When you arrive, a simple scan reveals the system's star is unremarkable, nearing the end of its main sequence phase. Among the planets orbiting it, one is in the habitable zone and shows clear signs of life, covered in dark green spots under the clouds.`

Author: I feel like this is unnecessarily loquacious :P

> 			`The trip to Zubenelhakrabi is completely normal and calm. Given everyone's trepidating expectations though, the trip's uneventfulness only manages to keep everyone on edge. When you arrive, a simple scan reveals the system's star is unremarkable, nearing the end of its main sequence phase. Among the planets orbiting it, one is in the habitable zone and shows clear signs of life, covered in dark green spots under the clouds.`

Reviewer: Lol yeah, the word trepidatious might be a bit much, but I only changed it because trepidating isn't actually a word and I wanted to keep my change as close to your original idea as possible.

> 			`The trip to Zubenelhakrabi is completely normal and calm. Given everyone's trepidating expectations though, the trip's uneventfulness only manages to keep everyone on edge. When you arrive, a simple scan reveals the system's star is unremarkable, nearing the end of its main sequence phase. Among the planets orbiting it, one is in the habitable zone and shows clear signs of life, covered in dark green spots under the clouds.`

Reviewer 2: Yeah, "trepidating" isn't a word, so something needs to change here.

> 			`The trip to Zubenelhakrabi is completely normal and calm. Given everyone's trepidating expectations though, the trip's uneventfulness only manages to keep everyone on edge. When you arrive, a simple scan reveals the system's star is unremarkable, nearing the end of its main sequence phase. Among the planets orbiting it, one is in the habitable zone and shows clear signs of life, covered in dark green spots under the clouds.`

Author: Oh, I was pretty sure it was. Is trepidatious a word? I'm sad i don't have a license to invent new words like Shakespeare :P

> 			`The trip to Zubenelhakrabi is completely normal and calm. Given everyone's trepidating expectations though, the trip's uneventfulness only manages to keep everyone on edge. When you arrive, a simple scan reveals the system's star is unremarkable, nearing the end of its main sequence phase. Among the planets orbiting it, one is in the habitable zone and shows clear signs of life, covered in dark green spots under the clouds.`

Reviewer 2: >Is trepidatious a word?

Yes. 
https://www.dictionary.com/browse/trepidatious (Redirected to "trepidation." Scroll down to "other words from trepidation." See "trepidatious.")

> 			`The trip to Zubenelhakrabi is completely normal and calm. Given everyone's trepidating expectations though, the trip's uneventfulness only manages to keep everyone on edge. When you arrive, a simple scan reveals the system's star is unremarkable, nearing the end of its main sequence phase. Among the planets orbiting it, one is in the habitable zone and shows clear signs of life, covered in dark green spots under the clouds.`

Reviewer 3: Why would you not just say "everyone's trepidation? Also, do not put a comma before and after though. Try this:
```suggestion
			`The trip to Zubenelhakrabi is completely uneventful. However, everyone's trepidation is such that the trip's uneventfulness only manages to keep everyone on edge. When you arrive, a simple scan reveals the system's star is unremarkable, nearing the end of its main sequence phase. Among the planets orbiting it, one is in the habitable zone and shows clear signs of life, covered in dark green spots under the clouds.`
```

				goto next
			label cannon
			`	"That ship is just a relic. The weapon that perished the Sheragi was indiscriminate. An irrational decision they made themselves."`
			label next
			`	As that is said, you're overcome by glimpses of memories that are not yours, not clear enough for you to identify what exactly is happening, but you can feel the sadness in them.`
			`	The voice continues: "I know what you seek here. The trinket you covet is of little relevance, even for your own, human standards. But if you wish to possess it, then take it. Perhaps the sight of that vessel will remind the living races of those who have come and gone... and how, despite their great accomplishments, their decision to make war reduced them to graveyards, relics, and memories."`
			`	The voice, and the overwhelming emotions that came alongside it, disappear as mysteriously as they came. Desyo, who witnessed you speaking into thin air, asks whether you feel okay. It seems like no one else heard it.`

	npc board
		government "Sheragi"
		personality derelict mute
		system Zubenelhakrabi
		ship "Emerald Sword" "Emerald Sword"
		conversation

> 			`You dock with the Emerald Sword, and the crew starts working to carefully find a way in, directed by the Hai archaeologists. There is no atmosphere inside it as the life support has long since ceased to work, and it takes a long time to restore it through your ship's own life support and patch tiny micrometeorite holes. Once it's in breathable conditions, it still smells like a tomb. You find many Sheragi skeletons and mummified remains. A few of them have prosthetics that survived better than their host. Your crew gets to work cleaning up the thick layer of dust and grime that covers every surface, while the more specialized crew tries to bring the power core back to life and take advantage of the zero-G environment to move the jump drive into the ship more easily. You'll need to land to properly install it, however. All that is left for you to do is relocate the many corpses littering the living spaces with your crew.`

Reviewer 2: I feel like you could really extend this conversation to make it more impactful. After having the Archon speak to you, I feel like it'd be fitting to linger on the skeletons here for much longer. Perhaps you have the option to wander the ship and observe the bodies; I imagine there are stories to be told by the positions that the Sheragi died in. And during that time the player could perhaps speak with Foster and ask if what you're doing is wrong; you're basically tomb raiding right now, after all.

Author: Interesting idea!

Reviewer 3: Seconded, huge potential for emotionally impactful narrative.

	
	on complete
		conversation

> 			`It takes some time to find a suitable landing spot - high enough to escape the highest concentrations of chlorine gas in the atmosphere but flat enough to land on. You eventually park your ships over a mesa. The environmental suits brought along "just in case" turn out to be very useful here. The atmosphere would be otherwise unbreathable for long periods of time.`

Reviewer 2: The "chlorine gas in the atmosphere" plot point seems to just be tossed into this conversation. Going off of my previous comment of making this feel more impactful, I'd perhaps change this to something along the lines of,
```suggestion
			`As you enter the atmosphere of the alien world, you begin running some scans of the air quality to determine if it is safe to breathe. You quickly learn why it is that the Sheragi perished in space; the atmosphere has a high concentration of chlorine gas, fatal to all living creatures aside from those that seem to have miraculously evolved here. It takes some time to find a suitable landing spot - high enough to escape the highest concentrations of chlorine gas in the atmosphere, but flat enough to land on. You eventually park your ships over a mesa. The environmental suits brought along "just in case" turn out to be very useful here. The atmosphere would be otherwise unbreathable for long periods of time.`
```

			`	Over the course of the next day, the workers unload some outfitting and repairing equipment, while scientists slowly work out the functions of the ship, and brief you and your crew on how to operate it. One of the workers invite you to visit the temporary outfitter and oversee the proper installation of the Outfits Expansion and the Jump Drive in the Emerald Sword.`
			`[You need to remove the Damaged Sheragi Systems to repair it.]`

mission "Emerald Sword Epilogue"
	name "The Emerald Sword"

> 	description "Return to Zug"

Reviewer 2: ```suggestion
	description "Return to Zug."
```

	landing
	source "Echoes of Tragedy"
	destination "Zug"
	passengers 20
	cargo "Outfitting Equipment" 25
	
	to offer
		has "The Emerald Sword 3: done"
	

> 	blocked `You need more bunks or cargo.`

Reviewer 2: I believe it's (nearly) impossible for this mission to be blocked given that the previous mission had higher requirements, so this blocked dialog isn't necessary.

Author: Big oops, i left that as a debug message because that mission kept failing to get offered when i was making it.

	
	on complete
		event "temporary outfitter removed"
		conversation

> 			`Your exotic new ship draws a lot of attention from the dockworkers, probably even more so due to the unprecedentedly large cannon. Albert and the Hai archaeologists spend most of the day examining your ship in more detail and recovering the corpses now that it's in a proper spaceport and they can access more complex equipment. Everything is completed in one particularly busy afternoon, and as night falls, the immense scale of the inside of the ship allows more floors to be installed, giving you even more living space.`

Reviewer 2: Glad to hear they didn't just dump the corpses on that planet.

> 			`	At that point, you go back to the same park as before with Albert and the three Hai in partial disguise, ice creams in hand and paws to commemorate the new acquisition.`

Reviewer 2: ```suggestion
			`	You go back to the same park as before with Albert and the three Hai in partial disguise, ice creams in hand and paws to commemorate the new acquisition.`
```

			`	"That is one impressive ship," you mention.`
			`	"True," Desyo says. "I've never seen a cannon that big before."`
			choice

> 				`	"That's what she said."`

Reviewer 2: Golden. 👌

			`	"So, the Sheragi apparently died off in a war of catastrophic proportions. One of our interns suggested we take a closer look underground, to check for military facilities or bunkers, even though our previous scans didn't show anything interesting, so we left her to play with the ground penetrating radar. and wouldn't you believe, there were some angled pieces of metal under the earth. Clearly man made. Or well, Sheragi-made in this case. It was a bunker, that apparently got hit by a powerful kinetic weapon, but we could still salvage some artifacts from the debris. Particularly, this thing."`
			`	Albert opens the safe, and shows you a weathered metal cube, around 40 cm on each side. he struggles with the weight, but manages to set it down on the floor.`
			`	"This appears to be a data storage device of some sort. It was connected to what we think were once computers, but nothing else is salvageable after spending thousands of years under wet dirt. Based on how sturdy this thing is, it's plausible that some of the data could have survived, like a time capsule."`

> 			`	"I have a contact back on vinci who could probably figure this out, and this is where you come in. You remember John, right? I can't leave the dig site, but I also don't have anyone else I could trust to carry such a valuable artifact. Will you take it there for me?"`

Author: Albert is distrustful of the Quarg. Wouldn't want them limiting his discoveries.

			choice
				`	"Sure, I'll take the cube to him."`
					goto take

> 				`	"I'm sorry, but that sounds sketchier than all the other jobs I did for you. The quarg were clear in saying that some things are not meant to be found."`

Author: It is because this is presumably something the Quarg wouldn't want humans to find. Getting in trouble with the Quarg would be an order of magnitude worse than getting in trouble with the Deep.

			choice
				`	"The bad news."`
					goto bad
				`	"The worse news."`
					goto worse
			label bad
			`	"The bad news is, this is a lot more complicated than I thought. I found out that the data in the box has survived, and can still be recovered after all this time, but it will need some specialized hardware and knowledge, and we don't have that yet."`
			`	"And the worse news?" You ask.`
			`	Well... Some people invaded my lab during the night, and stole a lot of the things I was working on. Thankfully my insurance covers most of it, and I still have most of my heavy expensive equipment, but the box... is gone.`
				goto whatnow
			label worse
			`	"Well... Some people invaded my lab during the night, and stole a lot of the things I was working on. Thankfully my insurance covers most of it, and I still have most of my heavy expensive equipment, but the box... is gone.`
			`	"And the bad news?"`
			`	"The bad news is, this is a lot more complicated than I thought. I found out that the data in the box has survived, and can still be recovered after all this time, but it will need some specialized hardware and knowledge, and we don't have that yet."`
			label whatnow

> 			`	"What do we do now? Do you have any idea where to find the box? Did you contact law enforcement?"

Reviewer 4: Needs a backtick at the end of the line here.

I pushed a PR to your master with the missing backtick for this and the one in line 439, just to make an already easy task even easier.

> 			`	"I did what I could, but if this was mandated as I expect, the police isn't going to help, and the box is probably off-world by now. However... if you have some cash at hand, and you're willing to go the extra mile to recover the box, there is a way."`

Author: That's more personal distrust than actual fact, i'd say. Outside factions might be willing to help, but might also impose limits on their work.

					goto haggle
				`	"How about you tell me what I want and I don't blow up your ship and your pirate friends?"`
					goto threat
				`	"Here's the money."`
					goto pay
			label nomoney
			`	"I know you don't. I can arrange for the Payment to be made in different ways. Organ donation, prostitution, voluntarizing as a test subject... We'll need an appraisal first though."`
			`	You politely decline.`
			`	"Come back to me when you have it, then, and I shall provide you with the information. We have all the time in the universe. Until you die, or someone decides to destroy the cube, that is."`
			`	They gesture, and the men promptly and diligently escort you back out to your ship.`
				defer
			label haggle
			`	Mammon doesn't look pleased.`
			`	"What part of non-negotiable did you not understand? You're pushing your luck, captain <last>."`
				goto question
			label threat
			`	Mammon just smiles, not at all intimidated.`
			`	"You think you can intimidate me? It's your funeral, human. If you were smart, you would know that my power far exceeds yours. You know I can't let you leave here alive now. It would set a bad precedent for me. Safety is my top priority."`

> 			`	You look around, and the men in suit are pointing guns at you. Alien guns. Before you can react, Mammon shoots you from under the table, fatally injuring your abdomen. Your half conscious body is dragged out of the ship, and your fleet being blown out of the sky by alien weapons is the last thing you see.`

Reviewer 5: This text assumes the player has a fleet at this point, but the player might just be flying alone or might have lost his/her fleet in the previous battle. Maybe change to "your ship being shredded to pieces by alien weapons is the last thing you see"

				die
			label pay
			`	Mammon takes the credit chips, examining them.`
			`	"Very good. The artifact you're looking for has been taken by some mercenaries. It seems to me that your friend messed with the wrong people. But that is not the information you paid for."`
			`	They lift their hands to the holographic map of the galaxy in the ceiling. It zooms in, filling up the entire room with stars. In front of you, you recognize the border betwen the Deep and the Dirt Belt.`
			`	"They have kept the artifact for themselves to sell on the Black market, and their base of operations is right here in Algieba."`
			`	The hologram zooms further in, on Watcher, and they point at it.`
			`	"You will find them in this uninhabited moon. They are hiding out in this forest, and the artifact is with them. Expect hostilities."`

> 			`	You thank Mammon for their services and get up to leave.`

Author: In that mission, i used "they" as a genderless pronoun, since it seems to be what most people agree on. But yes, substituting some of them for Mammon is a good idea.

			`	"Farewell, captain <last>. Remember: I am always watching."`
				accept
	
	on defer
		set visited
	
	on accept
		set "Paid Mammon"
		fail "The Box 3"

mission "Mammon Payment"        

> 	name "Mammon Payment"        

Author: If the payment is on accept, then due to how the game works, the mission would never get offered in the first place if you have less than 10M credits. I had to work around that.

> 	name "Mammon Payment"        

Author: I think i've tried this before and it didn't work. I'll check if it works.

> 	name "Mammon Payment"        

Author: Confirmed, it doesn't work. This has to be on a separate mission or else it doesn't offer.

			confusion 20
		system "Algieba"
		fleet "Marauder fleet X"
		dialog
			`You managed to destroy the entire mercenary fleet. You should have no problem landing on their camp now and recovering the cube.`
	
	on visit
		conversation
			`You attempt to evade the mercenary ships and make a quick assault on their encampment, but the ships follow you through the atmosphere, and ground personnel start shooting at you. Bombing it on a pass means you risk destroying the cube. You have no choice but to go back to space and eliminate the mercenary fleet.`
				launch
	
mission "The Box 6a"
	landing
	name "The box"
	source "Watcher"
	destination "Vinci"
	invisible
	
	to offer
		has "The Box 5: done"
	
	on offer
		conversation
			`You come in for a landing, and the few mercenaries that stayed on the ground march to your ship carrying a makeshift white flag. Dozens of men and women, tough looking and in uniform, carrying weapons but remaining peaceful and organized. These weren't ordinary criminals, it seems.`
			`	"We know when we lost" One of them yells to your ship on a megaphone. "What is it that you want?"`
			choice
				`(Shoot them)`
					goto shoot
				`(Talk to them)`
			

> 			`	You land and come out with your crew, forming your own defensive line.`

Reviewer 5: The player might not have a crew at this point (might be flying a smaller ship alone).

Author: I considered it cafully, but 1, there is no way to check whether the player has a crew or not, and 2, it's slightly unlikely that the player will be flying solo in a single-person ship at this point, since the last mission has you defeat an entire fleet of pirates.
The player could be piloting a kor automaton, but even those take crew if you want to use turrets.

			`	"Who do you work for? Where is the cube you stole?" You ask.`
			`	"So that cube was yours?" One woman up front says. "I knew that mission was sketchy... We're mercenaries. We were employed by some unknown men in suits who paid us really well to steal it, yet when we returned with it, they just scanned it and told us to dispose of it. Said that a digital copy should be enough, and the trouble should be enough retribution."`
			`	"We'll give you the cube on good terms if you spare us."`
			choice
				`(Spare them)`
					goto spare
				`(Open fire)`
					goto fire
			
			label shoot
			`	Without saying a word, you turn your ship towards them and open fire on the small army. Using ship mounted guns against personnel is extremely overkill, and they don't stand a chance to survive. There is a smoldering clearing around the place where they once stood, and a forest fire begins to spread.`
				goto next
			
			label fire

> 			`	You order your crew to open fire, and in seconds a small battle is raging. They weren't expecting you to attack, and that proves to be their demise. Many are dead before they can even react, and with the advantage of using your ships for cover, the rest are easily dealt with.`

Reviewer 5: As above, the player might not have a crew at this point.

			
			label next
			`	You get into the camp, shooting any mercenaries that remained hidden, and pillaging their tents. You find hundreds of thousands of credits worth of valuables, and eventually, you find the cube as well. With that in hand, you go back to your ship, ready to take it back to John.`
				decline
			
			label spare
			`	"Deal. Bring me the cube." You say.`
			`	While one of them goes fetch it, you wait in an uncomfortable silence, watching their every move, while they watch yours. Eventually the young man comes back and hands the cube to you. You examine it, confirming it is the artifact you were looking for.`

> 			`	"You have it, now go away. We won't get in your way again." She says, and the other mercenaries relax, going back to their tents.`

Author: Does it have to end with a comma even if there is no additional conversation after it?

			choice
				`	"They lived fast, died young, and left a good looking corpse"`
					goto approve
				`	"It's a pity they had to focus so much on conflict. They seemed very capable."`
			`	"Well, it's not like we can say much. At the time, we Hai were focusing a lot on conflict too. We even tried invading Sheragi lands. Perhaps if we were successful, they would have united against a common enemy and may have had survived their rebellious phase."`
				goto next
			label approve
			`	The Hai laughs in his own characteristic way.`
			`	"That's one way to put it. I can't deny the scale of their engineering were a marvel even for the Hai of that time, and our civilization was already old. Some things scale well with size, and they had no choice but to build big. But it's still a shame. Imagine such capability being used for peace."`
			label next
			`	"Anyway. It's rare to see a Human face here, even more intriguing for it to be staring at Sheragi artifacts."`
			`	You explain about the ruins recently discovered on Zug.`
			`	"Oh, right. They used to inhabit the part of Human space that is now called The Rim. Are you looking for anything in particular?"`

> 			`	"I would like to know more about Sheragi computers."

Reviewer 4: Needs a backtick at the end of the line here.

	
	to offer
		has "The box 7b: done"
	
	blocked `You arrive at the spaceport, but then you realize. Your ships don't have enough free bunks or cargo space required to bring the archaeologist team with you. You manage to contact them, and in an embarrassing conversation ask them to wait for you while you make sure you're ready.`
	
	on offer
		conversation
			`You meet up with the Hai archaeologists that will accompany you to Human space, an all female team. They introduce themselves to you.`
			`	Eyama is a linguist, specializing in the Sheragi language. She playfully greets you in their rather guttural tongue, with strange tones.`
			`	Rakei is an archaeological engineer, who immediately takes a liking to the cube as soon as she sees it, examining it's structure.`
			`	Finally, Desyo is a xenoprogramming expert, who studies alien software, and seemingly has a lot of experience with Sheragi code. Apparently, the displays of the Sheragi room of the museum all run in this exotic language, unseen by the usual public.`
			`	After the brief introduction, they each take their equipment and load them on the cargo bay of your ship. Rakei approaches you after everything is in place.`

> 			`	"We want to meet the Human archaeologist who uncovered the ruins in the rim. Can you take us there? We will start examining the cube along the way. I think by the time we get there, we'll have something to work with"`

Author: Rakei. It's stated in the previous line.

		conversation
			`It has been a long trip, almost halfway across the galaxy. The Hai archaeologists have been friendly to you, giving you updates every time they make a new breakthrough. During this time, the cube, previously thought to be inpenetrable, has been disassembled and cleaned. Components that decayed into a mess of oxidized wires and goopy paste over the eons have been identified and removed to substitute for equivalent ones, and a blueprint of the box and it's circuits has started to take shape.`
			`	You land, and make a quick call to Albert Foster, who doesn't believe you when you say you got alien archaeologists who are willing to help. You then look for the three Hai to figure out how they're going to go unnoticed in a Human world. You're surprised to find Rakei wearing long, dark but colorful baggy Human clothing with an equally oversized hoodie and a bandanna. You recognize this outfit, very popular with the youth these days, whom are generally regarded as delinquents by the previous generation. And she's carrying two more sets. It hides her body really well at the cost of making her look like a young boy, and the Hai's relatively short stature doesn't help.`

> 			`	"Call me miss deus ex machina, because I thought ahead and bought these from a merchant back home. Perfect disguise. The humans will never notice. How do I look?"`

Author: I think i messed up the names there, Eyama is the more culture-savvy of the three. The term has survived around 2000 years by now, what's another 1000? :P

			`	You can't disagree that it makes a good disguise, but refrain from telling them exactly how silly they would look in that clothing. Meanwhile the other two go back to their rooms to get dressed.`

> 			`	"Can you take us to Mr. Foster's museum? We'll take the cube and fill him in on the details."`

Author: Rakei (Or eyama, after i'm done with this update).
She's the only one left, since the other two went to get dressed.

		require "Jump Drive"
		conversation
			`You have a jump drive in your possession. Would you like to give it to Albert's crew?`
			choice
				`    (No.)`
					defer
				`    (Yes.)`
			`   You give Albert a call, and arrange to meet with him to plan the next step. He shows up at your ship, greets you, and starts loading in some equipment along with the jump drive you got. He has assembled a crew of around 20 people, including him and the three Hai archeologists, now comfortable to let go of their disguise once they're back in your ship.`
			`	"<first>, meet the people who will help us recover the Emerald Sword. I've just briefed them on the situation and what we plan to do. We're going to take some specialized tools with us, along with the Jump Drive, that will allow us to repair and outfit the ship in any world we can land on. I suggest you also hire some additional crew to man the ship once we capture it, since it's not going to pilot itself. If you're ready to go, Here are the coordinates to Zubenelhakrabi."`
			`	You thank albert, and make your preparations for the voyage into the unknown.`
				accept
	
	on accept
		fail "The Emerald Sword 2"
		outfit "Jump Drive" -1

> 		event "zubenelhakrabi discovered"

Author: Ideally, the ship would be landed on the planet and given to the player through mission functions. Being a derelict means i have to work around stuff to prevent the player from finding out about it before it's time, and many other things. Once that function is in the game, i plan to rewrite these last missions to make a lot more sense, and "unhide" that system.

> 		event "zubenelhakrabi discovered"

Author: the changes done through a quest seems to always apply only when you leave the planet. It's hardcoded. Either the system appears before you reach zug, or this "bug" happens. It would be nice to have an "apply" function to reload the data with new changes while still landed. this would allow some long multiple-ending planetary quests as well. I originally wanted to have a small adventure on watcher, but it's impossible without this apply thing

> 		event "zubenelhakrabi discovered"

Reviewer 2: I think that the system appearing seemingly out of no where for story purposes is fine. As for the system appearing far away if you look for it to quickly, that's just a quirk of the system. Nothing to be done about it.

> 		event "zubenelhakrabi discovered"

Reviewer 3: You could always add a little note to the player (Depart the planet to get updated map information.) - I've already seen one such note for the player's benefit in another quest so it's not without precedent.

A system appearing out of nowhere is perfectly fine. There are many more stars in that area of the galaxy than the hyperlanes lead to, and there's simply no reason to jump to any of the others unless you know something not widely advertised. It 'appearing' is actually kinda perfect in that sense.

			`	"That ship is just a relic. The weapon that perished the Sheragi was indiscriminate. An irrational decision they made themselves."`
			label next
			`	"The feeling in your chest becomes overpowering and you hold on to the command seat, gasping for air, light-headed. It seems evident those feelings are not your own, but are washing through you like a tidal wave. As you struggle to regain composure, the thoughts continue to form in your mind.`
			`	"I know what you seek here. The trinket you covet is of little relevance, even for your own, human standards. But if you will it, then take it. Perhaps the sight of that vessel will remind the feeble living races of those who have come and gone... and how, despite their great accomplishments, as they turn to war they are reduced to graveyards, relics, and memories"`
			`	"The voice, and the overwhelming emotions that came alongside it, disappear as mysteriously as they came. Desyo, who witnessed you speaking into thin air, asks whether you feel okay. It seems like no one else heard it."`

	npc board
		government "Sheragi"
		personality derelict mute
		system Zubenelhakrabi
		ship "Emerald Sword" "Emerald Sword"
		conversation

> 			`You dock with the Emerald Sword, and the crew starts working to carefully find a way in, directed by the Hai archaeologists. There is no atmosphere inside it, as the life support has long since ceased to work, and it takes a long time to restore it through your ship's life support and patch tiny micrometeorite holes. Once it's in breathable conditions, it still smells like a tomb. You find many Sheragi skeletons and mummified remains. A few of them have prosthetics that survived better than their host. Your crew gets to work relocating the remains to the cargo bay, and cleaning up the thick layer of dust and grime that covers every surface, while the more specialized crew tries to bring the power core back to life, and take advantage of the zero-G environment to move the jump drive into the ship more easily. You'll need to land to properly install it, however.`

Author: Which is what i think would happen if they were exposed to the vacuum, but i might be wrong, i'm not a biologist.

	
	on complete
		#outfit "Broken Ship Systems" -1

> 		#outfit "Refurbished Ship Systems" 1

Reviewer 5: Wouldn't it work to have just the removal of "Broken Systems" on landing in a separate mission? That mission should require the "Broken Systems" outfit to offer and just take it away.
Re-adding "Refurbished systems" doesn't seem necessary and setting up a temporary outfitter also doesn't seem necessary in such case.

Author: That is commented out because it's what i tried when i first made that mission. Apparently, the game doesn't check escorts for outfits to be removed and reinstalled, only the flagship and cargo. I was originally going to make the mission slightly longer, first making the ship flyable, but with a constant debuff, and then to remove that debuff, retrofitting it's internal space for a human crew.

	
	to offer
		has "Sheragi Archaeology: The Box 5: done"
	
	on offer
		conversation
			`You come in for a landing, and the few mercenaries that stayed on the ground march to your ship carrying a makeshift white flag. Dozens of men and women, tough looking and in uniform, unarmed, remaining peaceful and organized. These weren't ordinary criminals, it seems.`
			`	"We know when we've lost," one of them yells to your ship on a megaphone. "What is it that you want?"`
			choice
				`	(Shoot them.)`
					goto shoot
				`	(Talk to them.)`
			
			`	You land and cautiously open the hatch to talk with them.`
			`	"Who do you work for? Where is the cube you stole?" you ask.`
			`	"So that cube was yours?" One woman up front says. "I knew that mission was sketchy... We're mercenaries. We were employed by some men in suits who paid us really well to steal it, yet when we returned with it, they just scanned it and told us to dispose of it. Said that a digital copy should be enough, and the trouble should be enough retribution. We don't know who they were, we don't ask questions. That might be about to change now.`
			`	"Anyway, we'll give you the cube on good terms if you spare us."`
			choice

> 				`	(Spare them.)`

Reviewer 3: This whole bit here would be interesting if it either gave you or cost you some amount of Karma. Also, if you spare them, a positive relationship with a mercenary faction that could show up in future missions might be very interesting.

Author: A previous version of that quest had some karma cost involved, but people were against it since karma is only used for the main questline, and this is technically a sidequest. karma also only affects one mission in the end. maybe if the system was more fleshed out, it would be good, but people would need to agree on some stuff. It's a great discussion to have.

	
	on offer
		conversation
			`You arrive on Zug, and after contacting Albert and the three Hai archaeologists and following their direction, you find them in an ice cream shop. They're still wearing the youngster clothes, but not even bothering to hide their faces with the bandanna anymore. Surprisingly, nobody seems too distressed at the sight. Rakei waves for you and stands up, coming to you with ice creams in paw.`
			`	"Hello <first>! It seems our disguise worked better than intended. The humans around here think we're wearing a costume. Apparently young humans like to dress up as aliens?" she says confusedly.`
			`	"I had a feeling we'd fit right in with the furry crowd." Eyama responds proudly.`
			`	"It has been an interesting cultural exchange. Human space might be a great vacation spot for us. But we're not on vacation yet. We have a mystery to solve," she continues.`
			`	You sit down with your group on a bench in a nearby park, mostly empty, and Desyo picks up the conversation.`
			`	"So, the files in the black box were thankfully standard for Sheragi. Not very difficult to decode, once we managed to read them.`
			choice
				`	"What did it contain?"`

> 			`	"It was all fleet movement data and military strategy details. This system was apparently on the border of one of the faction's territories. On the final moments of the recording, a large transport fleet had entered the system, unarmed ships. They were followed by military vessels in pursuit. There's no telling what they thought at the time, but a military fleet was immediately dispatched to eliminate both fleets. One particular ship, which Eyama has translated the name as 'Emerald Sword,' started behaving erratically when given orders. It fired and missed a few shots until it eventually stopped and fled. Clear signs of a mutiny. However, that ship fled to a specific direction, away from any hyperlanes."`

Reviewer 3: ```suggestion
			`	"It was all fleet movement data and military strategy details. This system was apparently on the border of one of the faction's territories. On the final moments of the recording, a large transport fleet had entered the system, unarmed ships. They were followed by military vessels in pursuit. There's no telling what they thought at the time, but a military fleet was immediately dispatched to eliminate both fleets. One particular ship, which Eyama has translated the name as 'Emerald Sword,' started behaving erratically when given orders. It fired and missed a few shots until it eventually stopped and fled. Clear signs of a mutiny. However, what is perhaps most interesting is that when it fled, it fled in an unexpected direction, away from any hyperlanes."`
```

			`	Albert opens the safe, and shows you a weathered metal cube, around half a metre on each side. He struggles with the weight, but manages to set it down on the floor.`
			`	"This appears to be a data storage device of some sort. It was connected to what we think were once computers, but nothing else is salvageable after spending thousands of years under wet dirt. Based on how sturdy this thing is, it's plausible that some of the data could have survived, like a time capsule.`
			`	"I have a contact back on Vinci who could probably figure this out, and this is where you come in. You remember John, right? I can't leave the dig site, but I also don't have anyone else I could trust to carry such a valuable artifact. Will you take it there for me?"`
			choice

> 				`	"who's john again?"`

Reviewer 2: ```suggestion
				`	"Who's John again?"`
```
